doc/source/notebooks/advanced/multiclass_classification.py

```diff
@@ -64,8 +64,9 @@ def plot_from_samples(m, X, Y, samples, burn, thin):
     Fpred, Ypred = [], []
     num_samples = len(samples[0])
     for i in range(burn, num_samples, thin):
-        for var, var_samples in zip(m.trainable_variables, samples):
-            var.assign(var_samples[i])
+        for var, var_samples in zip(variables, samples):
+            v = var_samples[i]
+            var.unconstrained_variable.assign(v)
```

Comment:
this seems kinda subtle ... should the SamplingHelper not handle the conversion to constrained space and then simply assign that ?

Reply:
This is a function for plotting, that is weird by itself. And I'm not going to change it :)

Reply:
What I meant was, how you would have to handle the samples from the MCMC in general. This specific case is only highlighting that it might be a more general issue.

Reply:
could you not use sampling_helper.convert_constrained_values instead?
(NB- I think that's a bad name, it sounds like it expects to take in constrained values!)